BankApplication/Program.cs

```diff
@@ -95,30 +149,25 @@ private static void Put()
             Console.WriteLine("Enter account id: ");
             int id = Convert.ToInt32(Console.ReadLine());
 
-            // Put
+            _bank.PutAmount(new PutAccountParameters
+            {
+                Amount = sum,
+                Id = id
+            });
         }
 
         private static void CloseAccount()
         {
             Console.WriteLine("Enter the account id to close: ");
             int id = Convert.ToInt32(Console.ReadLine());
 
-            // Close
-        }
-
-        private static void NotifyAccountCreated(string message)
-        {
-            Console.WriteLine(message);
-
-        }
-
-        private static void NotifyAccountClosed(string message)
-        {
-            Console.WriteLine(message);
-
+            _bank.ClosedAccount(new CloseAccountParameters
+            {
+                Id = id
+            });
```

Comment:
для 1го или 2х параметров, отдельных класс выглядит излишним решением, проще его не создавать
```suggestion
            _bank.ClosedAccount(id);
```